src/System.Private.Xml/tests/XmlDocument/Performance/Perf.XmlDocument.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.IO;
```

Comment:
The perf tests in corefx were just deleted.  perf tests should now be added to the dotnet/performance repo.

Reply:
Seems better to remove/revert these tests